blint/binary.py

```diff
@@ -915,16 +915,24 @@ def add_elf_dynamic_entries(dynamic_entries, metadata):
             lief.ELF.DynamicEntry.TAG.RUNPATH,
             lief.ELF.DynamicEntry.TAG.RPATH,
         ]:
-            metadata["dynamic_entries"].append(
-                {
-                    "name": demangle_symbolic_name(entry.name),
-                    "tag": str(entry.tag).rsplit(".", maxsplit=1)[-1],
-                    "value": entry.value,
-                }
-            )
-            # Detect dotnet binary
-            if "netcoredeps" in entry.name:
-                metadata["exe_type"] = "dotnetbinary"
+            if hasattr(entry, 'name'):
+                metadata["dynamic_entries"].append(
+                    {
+                        "name": demangle_symbolic_name(entry.name),
+                        "tag": str(entry.tag).rsplit(".", maxsplit=1)[-1],
+                        "value": entry.value,
+                    }
+                )
+                if "netcoredeps" in entry.name:
+                    metadata["exe_type"] = "dotnetbinary"
+            elif hasattr(entry, 'runpath'):
+                metadata["dynamic_entries"].append(
+                    {
+                        "runpath": demangle_symbolic_name(entry.runpath),
```

Comment:
When debugging an entry I get name: `libfdk_acc.so`, and value: `318` when tag is `NEEDED`.
and, in case of `RUNPATH` I get:
name: `'$ORIGIN/'`
value: `375`